kubernetes/structures_pod.go

```diff
@@ -577,6 +581,28 @@ func flattenProjectedVolumeSource(in *v1.ProjectedVolumeSource) []interface{} {
 	return []interface{}{att}
 }
 
+func flattenEphemeralVolumeSource(in *v1.EphemeralVolumeSource) []interface{} {
+	att := make(map[string]interface{})
+	if in.VolumeClaimTemplate != nil {
```

Comment:
`in` is a pointer. Please check for nil before dereferencing it here.

Reply:
Several similar situations in the functions below as well.

Reply:
I'm not the original poster but I'm interested in this feature.
It seems that `in` cannot be nil because it wouldn't be called in the first place from `flattenVolumes`